### ISSUE TITLE
load library into boi error

### ISSUE THREAD

**Issue by sliverchimpmonk** [networkMe/missinghud2#45]
i keep getting an error message that says... [LoadLibraryIntoBoI] Error while executing LoadLibraryW in the BoI process. Try running the Missing HUD 2 from another directory path.
ive made sure i have the latest version, and ive also tried putting all the files in the boi folder, but it still wont work.

**Comment by babycthulhu96** [comment]
yeah im having the same issue. worked fine last night and when i went to get on today it popped up with the same issue.

**Comment by sliverchimpmonk** [comment]
i should also mention that its working on my desktop pc, but not on my laptop

**Comment by networkMe** [comment]
Often this error is fixed by running MHUD as admin. It's usually the result of something on your machine blocking DLL injection (as a security thing).

**Comment by sliverchimpmonk** [comment]
running it as an admin didnt work, so i tried just putting all the mhud files directly into the boi folder (which i also tried last night and it didnt work) but it did work when i ran it as an admin, and now works whether i run it as an admin or not
